### PR TITLE
[Vision] Add support for Xcode 14 beta 1,2 & 3

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: src/vision.cs [hunk 1]
```diff
@@ -74,6 +76,8 @@ enum VNImageCropAndScaleOption : ulong {
 		CenterCrop = 0,
 		ScaleFit = 1,
 		ScaleFill = 2,
+		ScaleFitRotate90CCW = 256 + ScaleFit,
```
Comment (dalexsoto) [inline src/vision.cs:79]:
```suggestion
		ScaleFitRotate90Ccw = 256 + ScaleFit,
```

File: src/vision.cs [hunk 2]
```diff
@@ -74,6 +76,8 @@ enum VNImageCropAndScaleOption : ulong {
...
+		ScaleFillRotate90CCW = 256 + ScaleFill,
```
Comment (dalexsoto) [inline src/vision.cs:80]:
```suggestion
		ScaleFillRotate90Ccw = 256 + ScaleFill,
```

File: src/vision.cs [hunk 3]
```diff
@@ -1837,6 +1854,10 @@ interface VNHorizonObservation {
 
 		[Export ("angle")]
 		nfloat Angle { get; }
+
+		[TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
+		[Export ("transformForImageWidth:height:")]
+		CGAffineTransform TransformForImageWidth (nuint width, nuint height);
```
Comment (dalexsoto) [inline src/vision.cs:1860]:
```suggestion
		CGAffineTransform CreateTransform (nuint width, nuint height);
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

**Review by dalexsoto — APPROVED** [review]

File: src/vision.cs [hunk 4]
```diff
@@ -3477,6 +3502,10 @@ interface VNGenerateOpticalFlowRequest {
 		[Static]
 		[Export ("currentRevision")]
 		VNGenerateOpticalFlowRequestRevision CurrentRevision { get; }
+
+		[TV (16,0), Mac (13,0), iOS (16,0)]
```
Comment (mandel-macaque) [inline src/vision.cs:3506]:
```suggestion
		[TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
```

File: src/vision.cs [hunk 5]
```diff
@@ -74,6 +76,8 @@ enum VNImageCropAndScaleOption : ulong {
 		CenterCrop = 0,
 		ScaleFit = 1,
 		ScaleFill = 2,
+		ScaleFitRotate90Ccw = 256 + ScaleFit,
```
Comment (mandel-macaque) [inline src/vision.cs:79]:
```suggestion
		[TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
		ScaleFitRotate90Ccw = 256 + ScaleFit,
```

File: src/vision.cs [hunk 6]
```diff
@@ -74,6 +76,8 @@ enum VNImageCropAndScaleOption : ulong {
...
+		ScaleFillRotate90Ccw = 256 + ScaleFill,
```
Comment (mandel-macaque) [inline src/vision.cs:80]:
```suggestion
		[TV (16,0), Mac (13,0), iOS (16,0), MacCatalyst (16,0)]
		ScaleFillRotate90Ccw = 256 + ScaleFill,
```

**Comment by mandel-macaque** [comment]
xtro tests failures are due to a mismatch with xcode14, which is an issue in the CI, landing and I'll be working on thet later.

**Pull request merged**